wolfcrypt/src/falcon.c

```diff
@@ -588,9 +588,22 @@ int wc_falcon_export_key(falcon_key* key, byte* priv, word32 *privSz,
  */
 int wc_falcon_check_key(falcon_key* key)
 {
-    /* Might want to try to sign and verify a random message here. */
+    /* Sign and verify a message. */
     int ret = 0;
-    (void)key;
+    int res = 0;
+    byte msg[] = "The wolfSSL team is here to make you ready for quantum computers!!";
+    word32 msglen = sizeof(msg);
```

Comment:
Also will produce cast warning. Sizeof returns size_t. You’ll need to cast to word32